src/rest/util.ts

```diff
@@ -9,6 +9,8 @@ import { User } from "../db/types";
 import { AuthenticationError } from "../errorClasses";
 import { getCollection } from "../db/mongoCollections";
 import { YesNoRating } from "types";
+import { Client, PlacesNearbyRanking, PlaceType1 } from "@googlemaps/google-maps-services-js";
```

Comment:
Is `PlaceType1` what google maps defines as restaurants? Just curious!

Reply:
https://developers.google.com/maps/documentation/places/web-service/supported_types#table1

`PlaceType1` is an enumeration of one kind of place type, which happens to include restaurants in this case